src/features/hotspots/root/HotspotsEmpty.tsx

```diff
@@ -3,13 +3,22 @@ import React from 'react'
 import { useTranslation } from 'react-i18next'
 import Box from '../../../components/Box'
 import Button from '../../../components/Button'
+import HotspotIcon from '../../../assets/images/blueHotspotIcon.svg'
+import Text from '../../../components/Text'
 import { RootNavigationProp } from '../../../navigation/main/tabTypes'
 
 const HotspotsEmpty = () => {
   const { t } = useTranslation()
   const navigation = useNavigation<RootNavigationProp>()
   return (
-    <Box justifyContent="flex-end" flex={1}>
+    <Box justifyContent="center" flex={1} padding="l">
+      <HotspotIcon />
+      <Text variant="h2" paddingVertical="l">
+        {t('hotspots.owned.title')}
+      </Text>
+      <Text variant="body2" paddingBottom="l" color="grayText" fontSize={20}>
```

Comment:
Could probably use `subtitleRegular` here in place of custom font size
` <Text variant="subtitleRegular" paddingBottom="l" color="grayText">`